app/controllers/api/AtomRepresentation.java

```diff
@@ -0,0 +1,160 @@
+        URI hashUri = curieResolver.resolve(new Curie(register.name() + "_hash", record.getHash()));
+
+        Map<String, String> keyValue = register.fields().stream()
+                .map(field -> new HashMap.SimpleEntry<String, String>(field.getName(), renderValue(record, field)))
+                .collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue));
+
+        String entry = "<entry>\n";
+        entry += "<id>urn:hash:" + record.getHash() + "</id>\n";
+        entry += "<title>" + hashUri.toString() + "</title>\n";
+        entry += "<updated>" + renderCreationTime(record) + "</updated>\n";
+        entry += "<author><name>openregister.org</name></author>\n";
+        entry += "<link href=\"" + hashUri.toString() + "\"></link>";
+        entry += "<content type=\"application/xml\">\n" + fieldEntriesForRecord(register, record) + "</content>\n";
+        entry += "</entry>\n";
+
+        return entry;
+    }
+
+    private String fieldEntriesForRecord(Register register, Record record) {
+        return register.fields().stream()
+                .map(f -> {
+                    String fieldOpenTag = "<" + FIELD_REGISTER_NAMESPACE_PREFIX + ":" + f.getName() + ">";
+                    String fieldCloseTag = "</" + FIELD_REGISTER_NAMESPACE_PREFIX + ":" + f.getName() + ">";
+                    String fieldValue = renderValue(record, f);
+                    return fieldOpenTag + fieldValue + fieldCloseTag;
+                })
+                .collect(Collectors.joining("\n")) + "\n";
+    }
+
+
+    private String renderValue(Record record, Field field) {
+        JsonNode jsonNode = record.getEntry().get(field.getName());
+        if (jsonNode == null) {
+            return "\"\"";
+        }
+        switch (field.getCardinality()) {
+            case ONE:
+                return renderScalar(field, jsonNode);
+            case MANY:
+                return renderList(field, jsonNode);
+            default:
+                throw new IllegalStateException("Invalid Cardinality: " + field.getCardinality());
+        }
+    }
+
+    private String renderScalar(Field field, JsonNode jsonNode) {
+        if (field.getRegister().isPresent()) {
+            Curie curie = new Curie(field.getRegister().get(), jsonNode.asText());
+            return String.format("%s", curieResolver.resolve(curie));
+        }
+        return StringEscapeUtils.escapeXml(jsonNode.asText());
+    }
+
+    private String renderList(Field field, JsonNode jsonNode) {
+        ArrayNode arrayNode = (ArrayNode) jsonNode;
+        return StreamSupport.stream(arrayNode.spliterator(), false)
+                .map(val -> renderScalar(field, val))
+                .collect(Collectors.joining(", "));
+    }
+
+    private String renderCreationTime(Record record) {
+        return record.getMetadata().isPresent()
+                ? record.getMetadata().get().creationTime.toString(RFC3339_DATETIME_FORMAT)
+                : null;
```

Comment:
I think we can assume that the Metadata always present with the record retrieved from database